pipelines/rj_smas__disparo_pic/tasks.py

```
@@ -1,276 +1,16 @@
 # -*- coding: utf-8 -*-
+# flake8: noqa:E501
```

Ruff (0.14.2)

Unused `noqa` directive (non-enabled: `E501`)

Remove unused `noqa` directive

(RUF100)